protocol/contracts/lease/src/lease/repay.rs

```diff
@@ -1,18 +1,18 @@
-use currencies::PaymentGroup;
 use sdk::cosmwasm_std::Timestamp;
 
+use currencies::PaymentGroup;
```

Comment:
please ensure any group usage within a contract goes through its `type XyzGroup = currencies::Xyz`. As we discussed, any usage of a group in a production contract code would refer to the local fir the contract typedefs